webapp/.babelrc

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
I'm just wondering if the should/can stay consistent with https://github.com/mattermost/mattermost-plugin-starter-template/blob/master/webapp/.babelrc here?

Reply:
You're right, and I missed. I've fixed 👍